src/llvm-ptls.cpp

```diff
@@ -70,6 +70,7 @@ void LowerPTLS::set_pgcstack_attrs(CallInst *pgcstack) const
 
 Instruction *LowerPTLS::emit_pgcstack_tp(Value *offset, Instruction *insertBefore) const
 {
+    IRBuilder<> builder(insertBefore);
     Value *tls;
     if (TargetTriple.isX86() && insertBefore->getFunction()->callsFunctionThatReturnsTwice()) {
         // Workaround LLVM bug by hiding the offset computation
```

Comment:
According to https://github.com/JuliaLang/julia/issues/17288 and https://bugs.llvm.org/show_bug.cgi?id=21183, maybe we don't need this workaround anymore? But I guess we can check that in a separate PR